app/assets/v2/js/metamask-approval.js

```diff
@@ -0,0 +1,16 @@
+disabled = [ '/', '/how/funder', '/how/contributor', '/tools' ];
+
+async function metamaskApproval() {
+  if (window.ethereum) {
+    window.web3 = new Web3(ethereum);
+    try {
+      if ($.inArray(window.location.pathname, disabled) == -1) {
+        await ethereum.enable();
```

Comment:
probably not a good idea to make this request on every page.. why not keep some state to let the system know when its enabled and when its not.